gpgs/src/java/com/defold/gpgs/GpgsJNI.java

```diff
@@ -171,6 +170,32 @@ private void sendSimpleMessage(int msg, String key_1, int value_1, String key_2,
         gpgsAddToQueue(msg, message);
     }
 
+    private void sendFailedMessage(int msg, String error_text, Exception e) {
+        if(e != null) {
+
+            if (e instanceof ApiException) {
+                ApiException apiException= (ApiException) e;
+                Integer errorStatusCode = apiException.getStatusCode();
+                error_text += ": " + GamesClientStatusCodes.getStatusCodeString(errorStatusCode) +" ("+errorStatusCode.toString()+")";
+
+                sendSimpleMessage(msg,
+                            "status", STATUS_FAILED,
+                            "error_status", errorStatusCode,
```

Comment:
Will this be used for logging purposes only or do we anticipate use of the error_status in game code? And if so, should we also expose the status codes as constants defined in gpgs.*?

I think these are the types of status codes you can get right?

https://developers.google.com/android/reference/com/google/android/gms/common/api/CommonStatusCodes

Reply:
Now we use codes for registration purposes. But, probably in the future, we will process codes in code too.
Which ones we will use is not yet known. Adding all codes ahead (as constants defined in gpgs) seems like overkill

Some of the codes are provided here - CommonStatusCodes (https://developers.google.com/android/reference/com/google/android/gms/common/api/CommonStatusCodes)
Another codes is here - GamesClientStatusCodes (https://developers.google.com/android/reference/com/google/android/gms/games/GamesClientStatusCodes.html)
And in other places like - SafetyNetStatusCodes (https://developers.google.com/android/reference/com/google/android/gms/safetynet/SafetyNetStatusCodes)

I propose to postpone the solution of this issue for later

Reply:
I agree. I was just curious to learn how it was supposed to be used now. Let's postpone until later.